src/app/profile/yourGiving/yourGiving.tpl.html

```diff
@@ -0,0 +1,140 @@
+                    <h3 class="m0 font-weight-mild">{{$ctrl.profile.yearToDate | currency:'$'}}</h3>
+                    <p translate>As of {{$ctrl.currentDate | date:'shortDate'}}</p>
+                  </div>
+                </div>
+                <div class="col-md-6">
+                  <div class="form-group">
+                    <label ng-if="$ctrl.profile.address" translate>Mailing Address:</label>
+                    <display-address address="$ctrl.profile.address"></display-address>
+                  </div>
+                  <div class="form-group mb0">
+                    <label ng-if="$ctrl.profile.phone" translate>Contact Phone:</label>
+                    <p class="mb0">{{$ctrl.profile.phone}}</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="col-md-4  col-sm-6 col-xs-12">
+          <div class="panel panel-full-height">
+            <div class="panel-body">
+              <!-- Giving Control -->
+              <a class="btn btn-block btn-primary" data-toggle="modal" data-target="#addPaymentMethodManageGiving" translate>Edit Your Recurring Gifts</a>
+              <a class="btn btn-block btn-primary mb- mt-" data-toggle="modal" data-target="#giveOneTime" translate>Give a 1-Time Gift</a>
+              <a class="btn btn-block btn-primary" data-toggle="modal" data-target="#whatTodDoManageGiving" translate>Stop or Restart a Gift</a>
+            </div>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="panel panel-margin-bottom" ng-if="!$ctrl.profileLoading">
+        <div class="panel-body">
+          <div class="row" style="position: relative;">
+            <div class="col-sm-12">
+              <h3 translate>Your Giving</h3>
+            </div>
+
+            <div class="col-xs-12 col-sm-12">
+              <div class="btn-toolbar">
+                <div class="btn-group" role="group" aria-label="...">
+                  <button type="button"
+                          class="btn btn-default"
+                          ng-class="{active: $ctrl.view == 'recipient'}"
+                          ng-click="$ctrl.setGivingView('recipient')">
+                    <span class="visible-xs" translate>By Recipient</span>
+                    <span class="hidden-xs" translate>Recipient View</span>
+                  </button>
+                  <button type="button"
+                          class="btn btn-default"
+                          ng-class="{active: $ctrl.view == 'monthly'}"
+                          ng-click="$ctrl.setGivingView('monthly')">
+                    <span class="visible-xs" translate>By Month</span>
+                    <span class="hidden-xs" translate>Monthly View</span>
+                  </button>
+                </div>
+
+                <div class="btn-group" uib-dropdown role="group" ng-if="$ctrl.view == 'recipient'">
```

Comment:
It's a little hard to tell from a user's standpoint whether this is a set of separate pages or a filter. The UX mockup has a button right aligned. Idk if this is that functionality. If it is a filter I think it could use the word Filter or Show or something next to it.